nowastecrawler/nowastecrawler/inserting_to_mongo.py

```diff
@@ -0,0 +1,39 @@
+import pymongo  # biblioteka mongo
+import pandas as pd  # python -m pip install pandas
```

Comment:
1. This dependency should be listed in requierments.txt dev file
2. Do we really need pandas only to read a JSON file? Python has a built-in module to it.